client/my-sites/site-settings/publishing-tools/index.jsx

```diff
@@ -105,32 +113,65 @@ class PublishingTools extends Component {
 		const formPending = this.isFormPending();
 
 		return (
-			<FormFieldset>
-				<SupportInfo
-					text={ translate(
-						'Allows you to publish new posts by sending an email to a special address.'
-					) }
-					link={
-						isAtomic
-							? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
-							: 'https://jetpack.com/support/post-by-email/'
-					}
-					privacyLink={ ! isAtomic }
-				/>
-				<JetpackModuleToggle
-					siteId={ selectedSiteId }
-					moduleSlug="post-by-email"
-					label={ translate( 'Publish posts by sending an email' ) }
-					disabled={ formPending || moduleUnavailable }
-				/>
+			<>
+				<FormFieldset>
+					<SupportInfo
+						text={ translate(
+							'Allows you to publish new posts by sending an email to a special address.'
+						) }
+						link={
+							isAtomic
+								? localizeUrl( 'https://wordpress.com/support/post-by-email/' )
+								: 'https://jetpack.com/support/post-by-email/'
+						}
+						privacyLink={ ! isAtomic }
+					/>
+					<JetpackModuleToggle
+						siteId={ selectedSiteId }
+						moduleSlug="post-by-email"
+						label={ translate( 'Publish posts by sending an email' ) }
+						disabled={ formPending || moduleUnavailable }
+					/>
+
+					{ this.renderPostByEmailSettings() }
+				</FormFieldset>
+				<hr />
+			</>
+		);
+	}
+
+	renderPostByVoiceModule() {
+		return (
+			<>
+				<FormFieldset>
+					<PostByVoiceSetting />
+				</FormFieldset>
+				<hr />
+			</>
```

Comment:
```suggestion
			<CompactCard className="site-settings__module-settings">
				<FormFieldset>
					<PostByVoiceSetting />
				</FormFieldset>
			</CompactCard>
```
Again, wrap in `CompactCard`